cmd/agent/subcommands/run/dependent_services_windows.go

```diff
@@ -53,8 +53,9 @@ var subservices = []Servicedef{
 	{
 		name: "sysprobe",
 		configKeys: map[string]config.Config{
-			"network_config.enabled":      config.SystemProbe,
-			"system_probe_config.enabled": config.SystemProbe,
+			"network_config.enabled":                         config.SystemProbe,
+			"system_probe_config.enabled":                    config.SystemProbe,
+			"system_probe_config.enable_windows_crash_check": config.SystemProbe,
```

Comment:
IMHO we don't need to keep things under `system_probe_config`, but you could do something like `windows_crash_check.enabled` instead

Reply:
the thinking was there's not additional config, so having an entire section seems excessive.  And, the OOM kill was enabled this way so it's consistent.

Reply:
There is no additional config...now 😎

Reply:
I'd consider the OOM kill way as legacy. All the other new products have their own namespaces, including the upcoming `ebpf` check.